Tasks/NugetPackager/NuGetPackager.ps1

```diff
@@ -44,6 +44,11 @@ if ($b_versionByBuild)
     Write-Verbose "Autoversion: Getting version number from build"
     ##Get Version from Build
     
+    if($Env:SYSTEM_HOSTTYPE -eq "release") {
+        Write-Error (Get-LocalizedString -Key "Autoversion: Getting version number from build option is not supported in releases")
```

Comment:
Yes, and it's the right thing to do in the short term.  I'm just pointing out that on principle more cases of this variance leads to a mine field of usage by the customer.  Long term, it feels like the right thing is for the build # used to flow into the release.